spec/models/category_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe Category do
+  let(:blank_values) { [nil, ""] }
+
+  describe "#name" do
+    it { should have_valid(:name).when("Streetlight", "Pothole") }
+    it { should_not have_valid(:name).when("#{'x' * 51}", *blank_values) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.